assets/css/style.scss

```diff
@@ -16,7 +16,7 @@ a {
 
 .tc39-box {
   border: transparent 1rem solid;
-  background-color: #ff9900;
+  background-color: #FA7C23;
```

Comment:
`#fc7c00` is the official color of the logo?

Reply:
thanks!

Reply:
I don't understand this `.tc39-box` class and div. Why don't we just reference the SVG file instead?